pitop/protoplus/adc.py

```diff
@@ -55,8 +55,8 @@ def poll(self, delay=0.5):
 
             results = self.read_all()
 
-            print("| " + str(results[0]) + "\t| " + str(results[1]) + "\t| " + str(results[2]) +
-                  "\t| " + str(results[3]) + "\t| " + str(results[4]) + "\t| " + str(results[5]) + "\t|")
+            print("| " + str(results[0]) + "\t| " + str(results[1]) + "\t| " + str(results[2])
```

Comment:
Any chance you can revert all the changes that aren't related to your code?

Reply:
Yeah sorry, was trying to get Flake8 checks working and made these changes - I'll revert